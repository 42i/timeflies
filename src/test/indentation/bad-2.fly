wp aa
    xxx
        yyyy
	bbb
    ccc
