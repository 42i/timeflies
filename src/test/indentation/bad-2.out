indentation/bad-2.fly:4: ERROR : work package indentation error
1 error.
